src/ray/gcs/format/gcs.fbs

```diff
@@ -108,7 +108,14 @@ table ResourcePair {
   value: double;
 }
 
+enum GcsTableNotificationMode:int {
+  CURRENT_VALUE = 0,
```

Comment:
It actually doesn't seem like we need a `CURRENT_VALUE` mode, since for logs and sets, this is basically the same thing as an `APPEND_OR_ADD` mode.

Reply:
Our internal codebase enabled redis reconnection and it will automatically re-call RequestNotifications to get latest data if recovered from disconnection. So the `CURRENT_VALUE` here is to indicate that the data is not newly added but current state. It can help callers to ignore the data or do some deduplication if necessary.
But since the community codebase doesn't have redis reconnection feature, it's indeed useless. I'll remove it.

Reply:
I see, thanks!